### ISSUE TITLE
[Bug]: PostgreSQL database backups don't work

### ISSUE THREAD

**Issue by unknownterritory** [coollabsio/coolify#4347]
### Error Message and Logs

There's nothing in the Executions list and nothing related to backups in the PostgreSQL logs. It seems like the backups are not triggered at all.

### Steps to Reproduce

1. Fresh install Coolify.
2. Create a Directus with Postgresql resource
3. Add a scheduled backup for the Postgresql database, either local or to a working S3 bucket elsewhere. Save.
4. Click on "Backup Now" for good measure.

### Coolify Version

v4.0.0-beta.370

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 24.04.1 LTS 

### Additional Information

Fresh installation of Coolify. Backups of Coolify itself to an object storage bucket on Scaleway are working just fine.
 Have two instances of Directus, both with PostgreSQL databases, on the same Hetzner server where Coolify is installed. Both instances are running fine and healthy. I've tried to set up backups for the databases (Postgresql (postgis/postgis:16-3.4-alpine)), both S3 enabled and local backups.

All the backups fail, both the scheduled and the on-demand backups (by pressing Backup Now). Stranger still is that nothing actually happens. No backup related logs or records in the Executions list.

**Comment by DMdeveloppement** [comment]
Just tested that the same issue append on Coolify CLoud

**Comment by DMdeveloppement** [comment]
It's maybe on postgis side. When i switch to a directus + postgres (without postgis) backups are working as expected

**Comment by peaklabs-dev** [comment]
Yes, postgis Images is not yet supported by Coolify Backups. It will be in the future once we have a full backup manager.
